categories.py

```diff
@@ -131,7 +130,7 @@
   "kshcheli":         ["proton-pog"],
   "kandrosov":        ["tau-pog"],
   "alebihan":         ["tau-pog"],
-  "mtosi":            ["tracking-pog"],
```

Comment:
oops, did not notice this change. @slava77 was this an intentional update?

Reply:
yes, it was intentional.
@mtosi @mmusich @vmariani 
may want to clarify if there were some longer term arrangements

Reply:
there have been no such arrangements.

Reply:
so all is good then